app/assets/javascripts/discourse/components/topic-list-item.js.es6

```diff
@@ -128,7 +129,7 @@ export default Ember.Component.extend(StringBuffer, {
     // highlight the last topic viewed
     if (this.session.get('lastTopicIdViewed') === this.get('topic.id')) {
       this.session.set('lastTopicIdViewed', null);
-      this.highlight();
+      this.highlight(true);
```

Comment:
I've grown to dislike booleans like this as parameters. Without looking at the signature it's not obvious what the `true` is for.

I think it would be better to add an opts object like:

`this.highlight({ isLastViewedTopic: true });`

(as an aside, if the function was like `setGreen(true)` I'd have no problem with it. It's obvious it means that it's `green` in that case!)